docs/specs/oidc.md

```diff
@@ -136,27 +144,26 @@ Refresh token is not used.
 
 ### response_type
 
-- `code`: Authorization Code Flow
-- `none`: [Authgear acting as authentication server with web application](#authgear-acting-as-authentication-server-with-web-application)
-
-### display
-
-The only supported value is `page`.
+- `none`: Nothing is included in the authentication response.
+- `code`: `code` is included in the authentication response. This is the authorization code flow.
 
 ### prompt
 
-The following values are supported.
+- `none`: An error is returned if the authentication request cannot be processed without user interaction.
+- `login`: Reauthenticate the end-user with any authenticator they have.
 
-- `login`
-- `none`
+> In the future, we may support select_account and content.
 
 ### max_age
 
-Unsupported.
+- `0`: It is equivalent to prompt=login.
+- Any positive integer: It is equivalent to prompt=login if the elapsed time of the last authentication is greater than n. Otherwise, it has no effect.
```

Comment:
What unit is n? Hours minutes seconds?

Reply:
[In seconds](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest). Let me be more specific here.